app/routes.py

```diff
@@ -1 +1,254 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint,jsonify, abort,make_response,request
+from app import db
+import requests
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime
+import os
+
+task_list_bp = Blueprint("tasks", __name__, url_prefix ="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix = "/goals")
+
+@task_list_bp.route("", methods = ["POST"])
+def create_tasks():
+    request_body = request.get_json()
+    print(request_body)
+    print("************")
+    
+    if (not "title" in request_body) or (not "description" in request_body):
+        return{
+            "details":"Invalid data"
+        }, 400
+    try:
+        new_task = Task.from_dict(request_body)
+        db.session.add(new_task)
+        db.session.commit()
+
+        #message = f"Task {new_task.title} successfully created"
+        return make_response(jsonify({"task":new_task.task_dict()}), 201)
+    
+    except KeyError as e:
+        abort(make_response("Invalid request. Missing required value: {e}"), 400)
```

Comment:
Nice error handling when processing a user-created request body! We should assume that a user's request body may contain errors!